fabtests/common/shared.c

```diff
@@ -1083,6 +1083,24 @@ int ft_complete_connect(struct fid_ep *ep, struct fid_eq *eq)
 	return 0;
 }
 
+int ft_verify_info(struct fi_info *fi_pep, struct fi_info *info)
+{
+	if (!info || !info->fabric_attr || !info->domain_attr ||
+	    !info->ep_attr || !info->tx_attr || !info->rx_attr)
+		return -FI_EINVAL;
+
+	if (!info->fabric_attr->prov_name ||
+	    !info->src_addr || !info->dest_addr ||
+	    info->fabric_attr->api_version != fi_pep->fabric_attr->api_version ||
+	    info->src_addr != fi_pep->src_addr ||
+	    info->dest_addr != fi_pep->dest_addr ||
+	    info->fabric_attr->name != fi_pep->fabric_attr->name ||
+	    info->domain_attr->name != fi_pep->domain_attr->name)
```

Comment:
Same -- remove.  I'm not sure the domains names are guaranteed to match.  Simply check that info->domain_attr->name is non-null.